### DISCUSSION TITLE
Add year of naming?

### DISCUSSION THREAD

**Discussion by ynux** [EqualStreetNames/equalstreetnames#821] [Ideas]
In a discussion with a historian about renaming a street it turned out that the year of the naming can be important. Specifically, if you want to rename a street in Germany, it will be easier if it was named during the Nazi regime.
I couldn't find a year_named tag in openstreetmap. Would it make sense to introduce it?
(Not an issue really, more of a question)

**Comment by jbelien** [comment]
Oh, yes, that's indeed an excellent idea. 👍 

I'll try to figure out if there is a way to do that in _OpenStreetMap_ tags.

**Comment by RobinLinde** [comment]
On [Taginfo](https://taginfo.openstreetmap.org/keys/name%3Astart_date), there are ~1400 uses for `name:start_date`, which seems like a suitable tag. Collecting this would also mean you would be able to see if there are changes in naming streets over time.

**Comment by jbelien** [comment]
There is (apparently) also https://wiki.openstreetmap.org/wiki/Proposed_features/Date_namespace

I also have "pinged" my "colleagues" from OpenStreetMap Belgium.

**Comment by seppesantens** [comment]
The name:start_date indeed seems suitable, but the date namespace @jbelien linked to is (better) documented, so I would go for that, if needed. Be sure to read on that same wiki-page about the scope of OSM. Personally, I'm not a fan of documenting history in OSM, I would suggest to do this in Wikidata on the Q-item of the street itself.

**Comment by jbelien** [comment]
> I would suggest to do this in Wikidata on the Q-item of the street itself

I thought about that solution too and I think it's indeed the best solution.
But it would require to create a Wikidata item for each street(name) ; that's already done for Brussels but I don't know if it's done for other cities.
(and my current process doesn't use the street(name) Wikidata item ... but that easily fixed)

**Comment by Marc-marc-marc** [comment]
name:start_date the start_date of the current name, it's imho the best for a road with only one old and one new name. this is useful for streets that have recently changed name, it avoids that someone makes the reverse correction with an old source (for example the old address of a shop on a website not up to date)

date namespace allow to store more than 2 names for ex name:2017=Rue de la frite name:2018=Rue de la fricandelle name=Rue du confinement :) but imho it'sn't the goal for osm, maybe more for openhistoricmap or wikidata

**Comment by RobinLinde** [comment]
> But it would require to create a Wikidata item for each street(name) ; that's already done for Brussels but I don't know if it's done for other cities.

Almost all Dutch streets have Wikidata items, so should be fine for NL
